OrthoEvol/Manager/db_mana_test.py

```diff
@@ -0,0 +1,49 @@
+from OrthoEvol.Manager.management import ProjectManagement
+from OrthoEvol.Manager.database_dispatcher import DatabaseDispatcher
+from OrthoEvol.Manager.config import yml
+from pkg_resources import resource_filename
+from pathlib import Path
+import yaml
+import getpass
+from datetime import datetime as d
+import os
+_jobname="jobname"
+
+# Set up project management
+pm_config_file = resource_filename(yml.__name__, "initialize_new.yml")
+with open(pm_config_file, 'r') as f:
+    pm_config = yaml.load(f, Loader=yaml.FullLoader)
+pm = ProjectManagement(**pm_config["Management_config"])
+
+# Set up database management
+db_config_file = resource_filename(yml.__name__, "databases.yml")
+with open(db_config_file, 'r') as f:
+    db_config = yaml.load(f, Loader=yaml.FullLoader)
+db_config.update(pm_config)
+db_config['Database_config']['Full']['NCBI']['NCBI_refseq_release']['upload_number'] = 12
+db_config['Database_config']['Full']['NCBI']['NCBI_refseq_release']['pbs_dict'] = {
```

Comment:
Could be better or not to create this dict in an outside file and import it. Or maybe it could go at the top of this file?

This way...users can access the default job parameters.

Just a suggestion tbh.

Reply:
Yes, this will be resolved whenever I work on the SGE module later.  To truly implement this right, it's going to take more work than I have time for.

Specifically, the YAML loading of `datetime` objects that have been formatted have made it necessary to turn this process into a funciton.

Reply:
> Yes, this will be resolved whenever I work on the SGE module later. To truly implement this right, it's going to take more work than I have time for.
> 
> Specifically, the YAML loading of `datetime` objects that have been formatted have made it necessary to turn this process into a funciton.

That's an interesting wrinkle about pyyaml.